src/sql/knexfile.js

```diff
@@ -3,6 +3,11 @@ const _ = require('lodash');
 const config = require('../config');
 const getConnectionParams = require('./getConnectionParams');
 
+// This was necessary because all of processingConfig is camelcase except for
+// absolute_theshold, which breaks the camelcase we had assumed
+// Consider migrating the property absolute_threshold to absoluteThreshold instead of doing this
+const processingConfigExceptions = ['methodSettings'];
```

Comment:
The solution proposed in the comment is probably better than what I did which is a fast patch, should I do that one instead? Some opinions on this pls. 
If we do it, it will probably not be too complicated.

In fact we could also make sure that anything stored as json is fully camelcase to avoid inconsistencies like this one creeping in

Reply:
This patch works, but it is definitely less clean than having absolute threshold in camel case. My opinion is that the best solution is to have absolute threshold in camel case and thus not have any exceptions. Exceptions can only be a cause of problems and create less readable code.

I would use this patch you coded in here if there are other fields like "absolute threshold" or if it would be too time consuming and complicated to fix the exceptions. Or if you just want to merge this PR and have this patch temporary and make absolute threshold to camel case in a separate PR.

Reply:
Overall, this patch is not too bad, it will work. Still, my opinion is that whenever we can have a cleaner solution with a reasonable amount of effort, I would choose the cleaner solution.

Reply:
ok, I'll try to do that then, I'll add a PR for the dynamoToSql script in iac, modify the pipeline, the ui and api, will leave links here when it is ready

Reply:
Unrelated to this PR, but I added [this](https://github.com/hms-dbmi-cellenics/iac/pull/405) to clean up the processing configs in production, it seems they still have the `api_url` and `auth_JWT` properties which were going to be removed in a previous ticket

Reply:
Also I added [this script](https://github.com/hms-dbmi-cellenics/iac/pull/406/files) to run the migration in dynamo to change to camelcase the processing config, I tested that v2 works correctly with a migrated experiment locally, but I will perform more testing on this script when it is time to run it (so when merging this PR and when running it production for the release too)